specification/src/main/asciidoc/platform/ServiceProviderInterface.adoc

```diff
@@ -1,64 +1,61 @@
+are packaged and integrated with any Jakarta EE product. Many types of
 service providers can be provided using the Connector API and packaging,
-including JDBC drivers, JMS providers, and JAXR providers. All Java EE
-products must support the Connector APIs, as specified in the Connector
-specification.
+including JDBC drivers, Jakarta Messaging providers, and Jakarta XML Registries
+providers. All Jakarta EE products must support the Connector APIs, as specified
+in the Connector specification.
 
-The Connector specification is available at
-_http://jcp.org/en/jsr/detail?id=322_ .
+=== Jakarta™ Authorization
 
-=== Java™ Authorization Service Provider Contract for Containers
+The Jakarta Authorization specification defines the contract
+between a Jakarta EE container and an authorization policy provider.
 
-The JACC specification defines the contract
-between a Java EE container and an authorization policy provider.
+=== Jakarta™ Transactions
 
-The JACC specification is available at
-_http://jcp.org/en/jsr/detail?id=115_ .
-
-=== Java™ Transaction API
-
-The Java Transaction API defines the
+The Jakarta Transactions defines the
 _TransactionSynchronizationRegistry_ interface that is intended for use
 by system level application server components such as persistence
-managers, resource adapters, as well as EJB and Web application
+managers, resource adapters, as well as Jakarta Enterprise Beans and Web application
 components. This provides the ability to register synchronization
 objects with special ordering semantics, associate resource objects with
 the current transaction, get the transaction context of the current
 transaction, get current transaction status, and mark the current
 transaction for rollback.
 
-The JTA specification is available at
-_http://jcp.org/en/jsr/detail?id=907_ .
+=== Jakarta™ Persistence
 
-=== Java™ Persistence
-
-Java Persistence provides interfaces in the
+Jakarta Persistence provides interfaces in the
 _javax.persistence.spi_ package that allow a persistence provider to be
-plugged into the Java Persistence framework.
-
-The Java Persistence specification is available
-at _http://jcp.org/en/jsr/detail?id=338_ .
+plugged into the Jakarta Persistence framework.
 
-=== JavaMail™
+=== Jakarta™ Mail
 
-The JavaMail specification describes how
-JavaMail protocol providers can be packaged and distributed so that they
-can be discovered and used through the JavaMail API. This allows the
-JavaMail API to be extended with support for new mail protocols and
+The Jakarta Mail specification describes how
+Jakarta Mail protocol providers can be packaged and distributed so that they
+can be discovered and used through the Jakarta Mail API. This allows the
+Jakarta Mail API to be extended with support for new mail protocols and
 mailbox formats.
 
-The JavaMail specification is available at
-_http://jcp.org/en/jsr/detail?id=919_ .
+=== Specification documents
+
+==== Original Java™ EE specification documents
```

Comment:
It will be cleaner and less confusing if we don't reference the JCP specs at all.
It would be better to omit these sections at the end and just depend on the
appendix that will link to all the specification documents.

Reply:
@bshannon ok, but I believe there are more sections that reference original JCP links. It would be better to do this in one PR for all documents after the new spec links are available.

Reply:
The expected form of spec links is described in [this message from Wayne](https://www.eclipse.org/lists/jakarta.ee-spec/msg00035.html).

Reply:
Would then make sense to update it right away? I saw the message after the PR was merged.